tests/simulations/ringtest/create_data.py

```diff
@@ -102,8 +104,8 @@ def make_edges(filename, edges, wanted_attributes):
     libsonata.EdgePopulation.write_indices(
         filename,
         name,
-        source_node_count=count,
-        target_node_count=count,
+        source_node_count=max(src_ids) + 1,
+        target_node_count=max(tgt_ids) + 1,
```

Comment:
This change is required for the gj edges file while I have only bidirectional connections 0<=>2, but id 1 has 0 connection. In this case, node_count should be 3 instead of 2. 
I see in [brainbuilder](https://github.com/openbraininstitute/brainbuilder/blob/d32007676e95e17e740e28ce7faae6716342bfaa/brainbuilder/utils/sonata/split_population.py#L238) the node_count is the max node ids + 1.
cc @mgeplf if you happen to be online.

Reply:
may you turn this comment in a comment in the code once it is settled (no question inside?). It will be helpful for future people that come here ans see a +1

Reply:
Done